airflow/bin/cli.py

```diff
@@ -72,15 +72,30 @@ def sigquit_handler(sig, frame):
     print("\n".join(code))
 
 
-def setup_logging(filename):
-    root = logging.getLogger()
+def setup_file_logging(filename, fmt=settings.SIMPLE_LOG_FORMAT):
     handler = logging.FileHandler(filename)
```

Comment:
If we are updating the way we do logging, we might want to make this configurable as well. e.g. Syslog could also be an option here and would probablt work better in some circumstances.